components/select/nz-select-top-control.component.html

```diff
@@ -21,7 +21,12 @@
       class="ant-select-selection-selected-value"
       [attr.title]="nzSelectService.listOfCachedSelectedOption[0]?.nzLabel"
       [ngStyle]="selectedValueStyle">
-      {{ nzSelectService.listOfCachedSelectedOption[0]?.nzLabel }}
+      <ng-container *ngIf="nzCustomTemplate; else noTemplate">
+        <ng-template [ngTemplateOutlet]="nzCustomTemplate" [ngTemplateOutletContext]="{ $implicit: nzSelectService.listOfCachedSelectedOption[0] }"></ng-template>
```

Comment:
nzCustomTemplate only support single mode?

Reply:
Yes, do we want to make it support the tag&multiple as well? cos I can't think of the use-cases for multi-select that needs custom template.

Reply:
nzCustomTemplate should support to replace both single and multiple modes